packages/@blockprotocol/type-system/src/index.ts

```diff
@@ -0,0 +1,9 @@
+import wasm from "../wasm/type-system_bg.wasm";
+import { setWasmInit } from "./common";
+
+export * from "../wasm/type-system";
+export { setWasmInit, TypeSystemInitializer } from "./common";
+
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-expect-error
```

Comment:
Honestly I left the reason out because I do not know the cause of it, or how to address it. I assume it is down to some growing pains with the WASM ecosystem.
<img width="1149" alt="image" src="https://user-images.githubusercontent.com/25749103/205324547-b1a8c853-1353-4206-b7e4-9260e4fafed8.png">

Spending time investigating the cause and coming up with a non-incorrect reason seemed like a bad investment when this package is still in its early days. I used `expect-error` as I hope this will be fixed in later tooling.

Reply:
Let's reflect that in the `--` comment. `"unknown"` is better than `undefined` here.